js/src/setup-mc/setup-stepper/saved-setup-stepper.js

```diff
@@ -58,6 +100,15 @@ const SavedSetupStepper = ( { savedStep, onRefetchSavedStep = () => {} } ) => {
 		}
 	};
 
+	function handleFormChange( errorMessage, newValue ) {
+		this( newValue ).catch( () => createNotice( 'error', errorMessage ) );
+	}
+
+	const initShippingRates = hasResolvedShippingRates ? shippingRates : null;
+	const initShippingTimes = hasResolvedShippingTimes ? shippingTimes : null;
+	const initTargetAudience = targetAudience?.location ? targetAudience : null;
+	const initSettings = settings?.shipping_rate ? settings : null;
```

Comment:
❓ 💅  I did some degugging. Settings is `null` by default until populate the object values... so maybe we can just pass `settings` directly ????

Reply:
Same for `initTargetAudience ` I guess

Reply:
It's waiting for completing the above processing to auto-save the default values for `settings`. Passing `settings` directly will lead to the `<Form>` receiving `initialValues` without those default values of `settings` when the `GET mc/settings` request is the last one to respond.

The problem can be simulated by adding delay to [fetchSettings](https://github.com/woocommerce/google-listings-and-ads/blob/43a2f76deda2218b5943be595bb6a1d0b1c5ee47/js/src/data/actions.js#L290):

```js
import { __unstableAwaitPromise } from '@wordpress/data-controls';

export function* fetchSettings() {
	yield __unstableAwaitPromise(
		new Promise( ( resolve ) => setTimeout( resolve, 3000 ) )
	);

	// ... omitted
}
```

deleting saved settings:

```sql
DELETE FROM wp_options WHERE option_name = 'gla_merchant_center';
```

and then refresh page to view the initial shipping options in step 2:

![2022-08-12 12 55 11](https://user-images.githubusercontent.com/17420811/184287554-004d2b2d-c31e-4a66-a31b-ff4a289bd6ad.png)

Reply:
Gotcha thanks.